src/main/java/dev/slimevr/bridge/PipeState.java

```diff
@@ -1,7 +1,5 @@
 package dev.slimevr.bridge;
 
 public enum PipeState {
-	CREATED,
-	OPEN,
-	ERROR
+	CREATED, OPEN, ERROR
```

Comment:
same here

Reply:
Eclipse always done it, if there are no arguments it joins the lines in enums